_drafts/gallery.md

```diff
@@ -28,7 +28,7 @@ image:
 #
 # Metainformation & Customization
 #
-meta_description:
+description:
```

Comment:
It does, and is [currently being improved](https://github.com/jekyll/jekyll-seo-tag/pull/151) beyond what's already there.